scripts/docker/alpine.3.6/Dockerfile

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+# Dockerfile that creates a container suitable to build dotnet-cli
+FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-20171119021156
+
+RUN apk add --no-cache sudo ncurses
+
+# Setup User to match Host User, and give superuser permissions
```

Comment:
Do we need this?  I thought we needed this to test our `.deb` packages in the docker container on Ubuntu, but I don't see why we need superuser permissions to do a "normal" build.

Reply:
I don't know, we do it in all Dockerfiles so I followed the suite. But you may be right. I'll try to see what happens if I remove it.

Reply:
I've verified it is not needed. The build completed ok with all of the user handling stuff disabled.